browser/ui/tabs/BUILD.gn

```diff
@@ -0,0 +1,29 @@
+# Copyright (c) 2025 The Brave Authors. All rights reserved.
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this file,
+# You can obtain one at https://mozilla.org/MPL/2.0/.
+
+assert(is_win || is_mac || is_linux || is_chromeos || is_android)
+
+# Note: Target names in this file should match those upstream where possible.
+
+source_set("tabs_public") {
```

Comment:
this is also not available on android so the assert above should be changed and the conditional below removed. Not sure why upstream left tabs_public out of `!is_android`, maybe they expected (as I did) that it would be used on both with different implementations

Reply:
If I do that the Android build will fail (because we `#define TabFeatures TabFeatures_Chromium`) and we won't be providing the `TabFeatures` header.

The build will still fail because there won't be an implementation available. I'd prefer to match what upstream is doing until they decide exactly how this is going to work.